docs/en/sql-reference/statements/drop.md

```diff
@@ -30,7 +30,7 @@ Also see [UNDROP TABLE](/docs/en/sql-reference/statements/undrop.md)
 Syntax:
 
 ``` sql
-DROP [TEMPORARY] TABLE [IF EXISTS] [IF EMPTY] [db.]name [ON CLUSTER cluster] [SYNC]
+DROP [TEMPORARY] TABLE [IF EXISTS] [IF EMPTY] [db.]name_1[,[db.]name_2][,[db.]name_3]... [ON CLUSTER cluster] [SYNC]
```

Comment:
This can be shorter: [db.]name_1[, [db.]name_2, ...]

Also, do the dropped tables need to be in the same schema/database? If no, then let's say `db_1` and `db_2` instead.

Reply:
Not too critical but wondering if there is some "atomicity" guarantee, i.e. if three tables are specified and the second one can't be dropped because it does not exist (or another error happens), will the third one still be dropped?

Reply:
About atomicity: A test for that would be nice (such that we at least know when the behavior changes). Consider also documenting the behavior.

Reply:
OK, added documentation and tests.